support-frontend/assets/components/paymentFrequencyTabs/paymentFrequencyTabsContainer.tsx

```diff
@@ -34,6 +33,12 @@ export function PaymentFrequencyTabsContainer({
 	);
 	const productType = useContributionsSelector(getContributionType);
 
+	// Force to monthly if hideOneOff, OneOff and refresh pressed
+	const contributionType = useContributionsSelector(getContributionType);
+	if (hideOneOff && contributionType === 'ONE_OFF') {
```

Comment:
@paul-daniel-dempsey I'm finding it hard to work out when this conditional will be `true`? Is this `if` block needed?